index.html

```diff
@@ -2802,13 +2826,13 @@ <h3>The <dfn>PropertyReadHandler</dfn> callback</h3>
             according to the <a>Protocol Bindings</a> and abort these steps.
           </li>
           <li>
-            Let |property| be the value of [[\td]]'s |properties|'s |name|.
+            Let |property| be the value of {{ExposedThing/[[td]]}}'s |properties|'s |name|.
             If it does not exist, send back a {{NotFoundError}} in the
             reply and abort these steps.
           </li>
           <li>
-            If there is an [[\unobserveHandler]] of type {{Function}} defined
-            for |name| on |property|, invoke that with |options|, send back a
+            If there is a user provided {{Function}} in {{ExposedThing/[[observeHandlers]]}} <a>internal slot</a>
```

Comment:
I think this should be {{ExposedThing/[[unobserveHandler]]}}

Reply:
Good catch, fixed!!